docs/dev/reference/hooks/modifyFrontendPage.md

```diff
@@ -25,7 +25,7 @@ insert tags, use the [`outputFrontendTemplate`](../outputFrontendTemplate) hook
 
     Content of the rendered front end template.
 
-2. *string* `$templateName`
+2. *string* `$template`
```

Comment:
Not sure about this. In the `parseTemplate` hook for example, there is a *Template* `$template`. I think that's why I called it `$templateName` here, since it is just the template name and not the `Template` instance.

Reply:
but two things:

1. everyone see it is string

Reply:
i think when you can see the type, it is clear that is not an object.

Reply:
I still think the naming is wrong, since it's not the template itself (neither the `Template` instance, nor the template _content_) but only the name of the template.

Reply:
otherwise we must name all string templates `$templateName`

Reply:
Yes, that's what should be done :).

Reply:
and here https://docs.contao.org/dev/reference/hooks/postDownload/ `$fileName`?

Reply:
Hm. I guess so. I mean, I think it is more important for `$template` vs. `$templateName`, because `string $template` could be misconstrued as the _contents_ of a template within the context of the previously mentioned hooks. At least I wouldn't call it `string $template` in my own code 😁